src/main/scala/com/cognite/sdk/scala/v1/resources/files.scala

```diff
@@ -39,16 +38,14 @@ class Files[F[_]](val requestSession: RequestSession[F])
               .mapResponse {
                 case Left(value) => throw value.error
                 case Right(Left(cdpApiError)) =>
-                  throw cdpApiError.asException(uri"$baseUri/byids")
+                  throw cdpApiError.asException(baseUri)
                 case Right(Right(value)) => Seq(value)
               }
           }
-      }
       case _ => throw new RuntimeException("Files only support creating one file per call")
     }
 
-  def uploadWithName(file: java.io.File, name: String): F[File] = {
-    val inputStream = new FileInputStream(file)
+  def upload(input: java.io.InputStream, name: String): F[File] = {
```

Comment:
Keep this as `uploadWithName` for now, and let `upload` use it similarly to what we had before.

Reply:
The problem was that there is no way to get the name from an inputstream. So then upload would still need to use java.io.File instead of an inputstream.